tests/base_init_test.php

```diff
@@ -92,7 +92,7 @@ public function testCallback3()
 
     public static function suite()
     {
-        return new PHPUnit\Framework\TestSuite("ezcBaseInitTest");
+        return new PHPUnit\Framework\TestSuite( "ezcBaseInitTest" );
```

Comment:
We never enforced CS in the `tests/` directory, so perhaps just exclude them? They seem like needless changes.